plugin_repackaging.sh

```
@@ -329,15 +329,31 @@ PY
 	echo "Step 3: Downloading dependencies"
 	echo "=========================================="
 	echo "Index URL: ${PIP_MIRROR_URL}"
-	[ -n "$PIP_PLATFORM" ] && echo "Platform: ${RAW_PLATFORM}"
+	[ -n "$PIP_PLATFORM_ARGS" ] && echo "Platform: ${RAW_PLATFORM}"
 
 	mkdir -p ./wheels
-	echo "Downloading wheels to ./wheels/..."
-	${PIP_CMD} download ${PIP_PLATFORM} --prefer-binary -r requirements.txt -d ./wheels \
+	echo "Downloading prebuilt wheels to ./wheels/..."
+	${PIP_CMD} download ${PIP_PLATFORM_ARGS} --only-binary=:all: --prefer-binary -r requirements.txt -d ./wheels \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)